src/components/section-layout/section.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
can you explain the purpose of this component and why viewport widths and heights are used

Reply:
There was a need for section component and I used it in almost all the places(And it is required). It is made to bind the content in the center of the page for larger screens. If you remove the section component then all the content will always be taking up 100% of the width of the page like the starter code had also for extra larger desktops. So this component is used to fix that issue.